src/plugins/chatwizard/chatwizard.js

```diff
@@ -475,17 +529,28 @@ var componentName = "wb-chtwzrd",
 	 * @param {String} title The title of the wizard window, as well as the notification
 	 */
 	buildChtwzrd = function( $selector, title ) {
-		$selector.after( "<div class='" + componentName + "-bubble-wrap'><a href='#" + componentName + "-container aria-controls='" + componentName + "-container class='" + componentName + "-link bubble trans-pulse' role='button'>" + i18nDict.trigger + "</a>" + ( !isNotif ? "<p class='trans-left'><span class='notif'>" + title +
-		"</span> <a href='#' class='notif-close' title='" + i18nDict.notification + "' aria-label='" + i18nDict.notification + "' role='button'>&times;</a></p>" : "" ) + "</div>" );
+		$selector.after( "<div class='" + componentName + "-bubble-wrap'><a href='#" + componentName + "-container' aria-controls='" + componentName + "-container' class='" + componentName + "-link bubble trans-pulse' role='button'>" + i18nDict.trigger + "</a>" + ( !isNotif ? "<p class='trans-left'><span class='notif'>" + title + "</span> <a href='#' class='notif-close' title='" + i18nDict.notification + "' aria-label='" + i18nDict.notification + "' role='button'>Ã—</a></p>" : "" ) + "</div>" );
 		$selector.next( selector + "-bubble-wrap" ).after( "<aside id='" + componentName + "-container' class='modal-content overlay-def " + componentName + "-container " + ( isInline ? " wb-chtwzrd-contained" : "" ) + "'></aside>" );
 
 		var $container = $( selector + "-container" );
-		$container.append( "<header class='modal-header header'><h2 class='modal-title title'>" + title + "</h2><button type='button' class='minimize' title='" + i18nDict.minimize + "'><span class='glyphicon glyphicon-chevron-down'></span></button></header>" );
+		$container.append(
+			"<header class='modal-header header'><h2 class='modal-title title'>" +
+			title + "</h2><button type='button' class='reset btn-reset' title='" +
+			i18nDict.reset +
+			"'> <span class='glyphicon glyphicon-refresh'></span></button>" +
+			"<button type='button' class='minimize' title='" +
+			i18nDict.minimize +
+			"'><span class='glyphicon glyphicon-chevron-down'></span></button></header>" );
 		$container.append( "<form class='modal-body body' method='GET'></form>" );
 
 		var $form = $( ".body", $container );
 		$form.append( "<div class='conversation' tabindex='0'><section class='history' aria-live='assertive'><h3 class='wb-inv'>" + i18nDict.conversation + "</h3></section><section class='reply'><h3 class='wb-inv'>" + i18nDict.reply + "</h3><div class='inputs-zone'></div></section><div class='form-params'></div></div>" );
```

Comment:
`tabindex="0"` is not needed.
```suggestion
		$form.append( "<div class='conversation'><section class='history' aria-live='assertive'><h3 class='wb-inv'>" + i18nDict.conversation + "</h3></section><section class='reply'><h3 class='wb-inv'>" + i18nDict.reply + "</h3><div class='inputs-zone'></div></section><div class='form-params'></div></div>" );
```